server/src/services/announcementService.ts

```diff
@@ -0,0 +1,47 @@
+import db from "../database/database";
+
+async function fetchAnnouncementInfo(announcementId) {
+    const query = "SELECT * FROM announcements WHERE id = $1";
+    const res = await db.query(query, [announcementId]);
+
+    const announcement = res.rows[0];
+    return announcement;
+}
+
+async function fetchAnnouncements() {
```

Comment:
I don't think this endpoint will ever have an actual usecase so maybe we should just remove it?